typings/lhr.d.ts

```diff
@@ -81,6 +81,23 @@ declare global {
         blockedUrlPatterns: string[];
         extraHeaders: Crdp.Network.Headers;
       }
+
+      export module Audit {
```

Comment:
curious why this is in lhr.d.ts instead of audit.d.ts

Reply:
> curious why this is in lhr.d.ts instead of audit.d.ts

Like `lhr-lite.d.ts`, it seems like all the LHR stuff should be defined in `lhr.d.ts`, instead of spread over `lhr.d.ts` and `audit.d.ts`. That would leave `audit.d.ts` with types internal to operating an audit, while `Audit.Result` stuff would all be in `lhr.d.t.s` (with a good chunk of that just inherited from `lhr-lite.d.ts`).

This starts on that path, plus is a nice dividing line between this finalized-ish `details` definition vs the handwavey/just-make-it-work definitions in `audit.d.ts`